CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # nypr-ui Changelog
 
+## 0.2.19
+- [BUGFIX] fix NYPR brand header mobile landscape vertical scroll https://jira.wnyc.org/browse/RT-941
+- [ENHANCEMENT] remove NYPR logo link
+
```

Comment:
I'll update this again with the NJPR logo addition, etc when we're ready for release.